walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,171 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void optionalAttendeeDisregarded() {
+    // Have three people have different events where one is an optional attendee busy all day. 
```

Comment:
Sweet diagrams and descriptions. People often read tests in order to understand how the actual code is supposed to work. This is one of those times; these tests really help me understand what you're trying to do in your code.

Reply:
Thank you Kenneth